Specs/Core/AxisAlignedBoundingBoxSpec.js

```diff
@@ -161,4 +161,31 @@ defineSuite([
             AxisAlignedBoundingBox.intersectPlane(box, undefined);
         }).toThrowDeveloperError();
     });
+
+    var scratchCartesian3 = new Cartesian3();
+    var secondScratchCartesian3 = new Cartesian3();
```

Comment:
In the future, don't worry about using scratch variables for tests, they're not needed.